src/mlpack/methods/lars/lars_main.cpp

```diff
@@ -149,11 +149,11 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
       false, "no results will be saved");
   ReportIgnoredParam(params, {{ "test", true }}, "output_predictions");
 
-  LARS* lars;
+  LARS<>* lars;
```

Comment:
For convenience, because `params.Get()` will return a pointer for a `PARAM_MODEL_IN` or `PARAM_MODEL_OUT`.  I suppose it could all be refactored, but it would be tedious and I'm not sure it's worth the effort.  (At least, not in this PR.)

Reply:
Okay, make sense to me, no need for refactoring :+1: